index.js

```diff
@@ -60,10 +82,10 @@ function addBooks() {
 
 function loadBooks() {
   JSON.parse(localStorage.getItem('booksArray')).forEach((book, i) => {
-    creaateAndAppend(book.title, book.author, i);
+    const newBook = new BookObject(book.title, book.author, i);
+    newBook.add();
+    theBooks.push(newBook);
   });
-
-  theBooks = JSON.parse(localStorage.getItem('booksArray'));
 }
```

Comment:
- [ ] Your methods for implementing the functionalities should be within the **BookObject** class. Kindly move this 
**`loadBooks()`** method into the class.